File: DarkOrbit 7.5.3/Net/netty/handlers/DisplaySettingsRequestHandler.cs
using Ow.Game;
using Ow.Managers;
using Ow.Net.netty.commands;
using Ow.Net.netty.requests;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Ow.Net.netty.handlers
{
    class DisplaySettingsRequestHandler : IHandler
    {
        public void execute(GameSession gameSession, byte[] bytes)
        {
            var read = new DisplaySettingsRequest();
            read.readCommand(bytes);

            var player = gameSession.Player;
            var displaySettings = player.Settings.Display;

            if (displaySettings.displayDrones != read.displayDrones)
                player.DroneManager.UpdateDrones();

            displaySettings.notSet = false;
            displaySettings.allowAutoQuality = read.useAutoQuality;
            displaySettings.displayBonusBoxes = read.displayBoxes;
            displaySettings.displayChat = read.displayChat;
            displaySettings.displayDrones = read.displayDrones;
            displaySettings.displayFreeCargoBoxes = read.displayCargoboxes;
            displaySettings.displayHitpointBubbles = read.displayHitpointBubbles;
            displaySettings.displayNotFreeCargoBoxes = read.displayPenaltyCargoboxes;
            displaySettings.displayNotifications = read.displayNotifications;
            displaySettings.displayPlayerNames = read.displayPlayerName;
            displaySettings.displayResources = read.displayResources;
            displaySettings.displayWindowsBackground = read.displayWindowBackground;
            displaySettings.dragWindowsAlways = read.alwaysDraggableWindows;
            displaySettings.hoverShips = read.shipHovering;
            displaySettings.preloadUserShips = read.preloadUserShips;
            displaySettings.showPremiumQuickslotBar = read.showSecondQuickslotBar;

            QueryManager.SavePlayer.Settings(player);
        }
    }
}

File: DarkOrbit 9.0/DarkOrbit 9.0 Emulator/src/simulator/netty/handlers/DisplaySettingsRequestHandler.java
package simulator.netty.handlers;

import mysql.QueryManager;
import net.game_server.GameServerClientConnection;
import net.game_server.GameSession;
import simulator.netty.ClientCommand;
import simulator.netty.clientCommands.DisplaySettingsRequest;
import simulator.users.ClientSettings;
import utils.Log;

/**
 Created by Pedro on 31-03-2015.
 */
public class DisplaySettingsRequestHandler
        implements ICommandHandler {

    private final GameServerClientConnection mGameServerClientConnection;
    private final DisplaySettingsRequest     mDisplaySettingsRequest;

    public DisplaySettingsRequestHandler(final GameServerClientConnection pGameServerClientConnection,
                                         final ClientCommand pDisplaySettingsRequest) {
        this.mGameServerClientConnection = pGameServerClientConnection;
        this.mDisplaySettingsRequest = (DisplaySettingsRequest) pDisplaySettingsRequest;
    }

    @Override
    public void execute() {
        Log.pt("DISPLAY SETTINGS REQUEST HANDLER");
        final GameSession gameSession = this.mGameServerClientConnection.getGameSession();
        if (gameSession != null) {
            final ClientSettings.Display display = gameSession.getAccount()
                                                              .getClientSettingsManager()
                                                              .getClientSettings()
                                                              .getDisplay();
            display.setNotSet(false);
            display.setDisplayNotFreeCargoBoxes(this.mDisplaySettingsRequest.displayNotFreeCargoBoxes);
            display.setDisplayResources(this.mDisplaySettingsRequest.displayResources);
            display.setDisplayWindowsBackground(this.mDisplaySettingsRequest.displayWindowsBackground);
            display.setDisplayBonusBoxes(this.mDisplaySettingsRequest.displayBonusBoxes);
            display.setDisplayNotifications(this.mDisplaySettingsRequest.displayNotifications);
            display.setShowPremiumQuickslotBar(this.mDisplaySettingsRequest.showPremiumQuickslotBar);
            display.setDisplayPlayerNames(this.mDisplaySettingsRequest.displayPlayerNames);
            display.setDragWindowsAlways(this.mDisplaySettingsRequest.dragWindowsAlways);
            display.setDisplayHitpointBubbles(this.mDisplaySettingsRequest.displayHitpointBubbles);
            display.setDisplayDrones(this.mDisplaySettingsRequest.displayDrones);
            display.setPreloadUserShips(this.mDisplaySettingsRequest.preloadUserShips);
            display.setHoverShips(this.mDisplaySettingsRequest.hoverShips);
            display.setAllowAutoQuality(this.mDisplaySettingsRequest.allowAutoQuality);
            display.setDisplayChat(this.mDisplaySettingsRequest.displayChat);
            display.setDisplayFreeCargoBoxes(this.mDisplaySettingsRequest.displayFreeCargoBoxes);
            display.setShowNotOwnedItems(this.mDisplaySettingsRequest.showNotOwnedItems);
            QueryManager.saveAccount(gameSession.getAccount());
        }
    }
}
